<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Time Tracker</string>
  </property>
  <widget class="QTableWidget" name="table_allGames">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>501</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addGame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_addTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>520</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Played Time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
